otter/test/convergence/test_steps.py

```diff
@@ -504,59 +506,16 @@ def test_remove_nodes_from_clb(self):
 
     def test_remove_nodes_from_clb_predicate(self):
         """
-        :obj:`RemoveNodesFromCLB` only accepts 202, 413, 400, and some 422
-        responses.  However, only 202, 413, and 422s are covered by this test.
-        400's will be covered by another test as they require retry.
+        :obj:`RemoveNodesFromCLB` only succeeds on 202.
         """
         lb_id = "12345"
         node_ids = [str(i) for i in range(5)]
         step = RemoveNodesFromCLB(lb_id=lb_id, node_ids=pset(node_ids))
         request = step.as_effect()
-
         self.assertTrue(request.intent.json_response)
-
         predicate = request.intent.success_pred
-
         self.assertTrue(predicate(StubResponse(202, {}), None))
-        self.assertTrue(predicate(StubResponse(413, {}), None))
-        self.assertTrue(predicate(
-            StubResponse(422, {}),
-            json.dumps({
-                "message": "The load balancer is deleted and considered "
-                           "immutable.",
-                "code": 422
-            })))
-        self.assertTrue(predicate(
-            StubResponse(422, {}),
-            json.dumps({
-                "message": "Load Balancer '12345' has a status of "
-                           "'PENDING_UPDATE' and is considered immutable.",
-                "code": 422
-            })))
-        self.assertTrue(predicate(
-            StubResponse(422, {}),
-            json.dumps({
-                "message": "Load Balancer '12345' has a status of "
-                           "'PENDING_DELETE' and is considered immutable.",
-                "code": 422
-            })))
-
-        self.assertFalse(predicate(StubResponse(404, {}), None))
-        self.assertFalse(predicate(
-            StubResponse(422, {}),
-            json.dumps({
-                "message": "Duplicate nodes detected. One or more "
-                           "nodes already configured on load "
-                           "balancer.",
-                "code": 422
-            })))
-        # This one is just malformed but similar to a good message.
-        self.assertFalse(predicate(
-            StubResponse(422, {}),
-            json.dumps({
-                "message": "The load balancer is considered immutable.",
-                "code": 422
-            })))
+        self.assertFalse(predicate(StubResponse(200, {}), None))
 
     def test_remove_nodes_from_clb_retry(self):
```

Comment:
This seems like a duplicate test to the cloud_client test `test_remove_clb_nodes_retry_on_some_invalid_nodes` - One of them can probably be removed, but it depends on where we want the retry to happen.